acceptance/redis32/main.go

```
@@ -0,0 +1,174 @@
+		"error": err.Error(),
+	})
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusInternalServerError)
+	w.Write(message)
+}
+
+func newPool(addr string, password string) *redis.Pool {
+	return &redis.Pool{
+		MaxIdle:     3,
+		MaxActive:   10,
+		IdleTimeout: 1 * time.Second,
+		Dial: func() (redis.Conn, error) {
+			c, err := redis.Dial("tcp", addr)
+			if err != nil {
+				return nil, err
+			}
+			if _, err := c.Do("AUTH", password); err != nil {
+				c.Close()
+				return nil, err
+			}
+			return c, nil
+		},
+		TestOnBorrow: func(c redis.Conn, t time.Time) error {
+			threshold := 5 * time.Second
+			if time.Since(t) < threshold {
+				return nil
+			}
+			_, err := c.Do("PING")
+			return err
+		},
+	}
+}
+
+func testSetGetDelete(w http.ResponseWriter, r *http.Request) {
+	client = pool.Get()
+	defer client.Close()
+	log.Printf("active connections: %d", pool.ActiveCount())
+	// Set and check value
+	_, err := client.Do("SET", "test", "test")
+	if err != nil {
+		writeError(w, err)
+		return
+	}
+
+	value, err := redis.String(client.Do("GET", "test"))
+	if err != nil {
+		writeError(w, err)
+		return
+	}
+	if value != "test" {
+		writeError(w, fmt.Errorf("incorrect value: %s", value))
+		return
+	}
+
+	_, err = client.Do("DEL", "test")
+	if err != nil {
+		writeError(w, err)
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+}
+
+func info(w http.ResponseWriter, r *http.Request) {
+	client = pool.Get()
+	defer client.Close()
+
+	parameter := r.URL.Query().Get("s")
+
+	if parameter == "" {
+		parameter = "all"
+	}
+
+	infoString, err := redis.String(client.Do("INFO", parameter))
+
+	if err != nil {
+		writeError(w, err)
+		return
+	}
+
+	infoMap := make(map[string]string)
+
+	for _, line := range strings.Split(infoString, "\r\n") {
+		part := strings.Split(line, ":")
+		if len(part) == 2 {
+			infoMap[string(part[0])] = string(part[1])
+		}
+	}
+
+	jresp, _ := json.Marshal(infoMap)
```

Don't ignore errors.